SeleniumPracticalExercise/TestCases/AddEmployee.cs

```diff
@@ -50,13 +42,15 @@ public void AddEmployeeTest()
             leftNavPanel.ClickPIM(); 
 
             // 8. Search for the employee you just created by Employee Id
-            employeeInformationPage.SetTxtBoxEmpID(storeEmployeeID); 
-
-            employeeInformationPage.SearchButton();  
+            employeeInformationPage.SetTxtBoxEmpID(storeEmployeeID);
 
             // 9. In the employee search results, use NUnit asserts to validate that Id, First Name, and Last Name are correct
-            employeeInformationPage.verifyEmployeeId(storeEmployeeID);
-            employeeInformationPage.verifyName(firstName, lastName);
-        }
-        
+            string actualIdResults = employeeInformationPage.verifyEmployeeId();
+            string actualFirstNameResults = employeeInformationPage.verifyFirstNameResults();
+            string actualLastNameResults = employeeInformationPage.verifyLasttNameResults();
+
+            Assert.AreEqual(storeEmployeeID, actualIdResults, "ID does not match expected value.");
```

Comment:
I don't think we've talked about this yet but we write assert comments as the action being taken, e.g. "Verify employee ID".

What you have isn't wrong... this is just another case where the automation team has agreed to do something a certain way for consistency.

Reply:
renamed them to have verify in it

Reply:
The entire assert comment should be, "Verify employee ID". Remove "does not match expected value." Follow this pattern for the other asserts as well.

Reply:
rename them to....
 Verify Employee ID.
 Verify First Name.
Verify Last Name.